debian/README.Debian

```diff
@@ -0,0 +1,6 @@
+distribution-gpg-keys for Debian
+----------------------------------------------------------
+
+<possible notes regarding this package - if none, delete this file>
```

Comment:
Can you please add here the guidelines on how to package the file? The one you put in PR comment.

Reply:
it's with debhelper if you want to test the build from my branch

```
sudo apt-get -y debhelper cdbs lintian build-essential fakeroot devscripts dh-make dput coreutils wget

wget https://github.com/xsuchy/distribution-gpg-keys/archive/refs/tags/distribution-gpg-keys-1.72-1.tar.gz -O distribution-gpg-keys_1.72.orig.tar.gz

tar -xvf distribution-gpg-keys_1.72.orig.tar.gz

cd distribution-gpg-keys*/

mkdir -p debian/source

wget https://github.com/amidevous/distribution-gpg-keys/raw/main/debian/source/format -O debian/source/format

wget https://github.com/amidevous/distribution-gpg-keys/raw/main/debian/README.Debian -O debian/README.Debian

wget https://github.com/amidevous/distribution-gpg-keys/raw/main/debian/changelog -O debian/changelog

wget https://github.com/amidevous/distribution-gpg-keys/raw/main/debian/control -O debian/control

wget https://github.com/amidevous/distribution-gpg-keys/raw/main/debian/copyright -O debian/copyright

wget https://github.com/amidevous/distribution-gpg-keys/raw/main/debian/rules -O debian/rules

debuild
```
if you have an error it is necessary to change the version number of debhelper because each debian or ubuntu system has its own version

it's a phenomenon that deban emptors know

they also know who should adapt the distribution's code

in the changelog here I have respected the rule because there is a rule

if you publish an example of compilation code as is the case here, you must always indicate unstable as the version code

and it is up to the person who compiles the package to change the code and adapt it to the distribution in which he publishes the package

whatever distribution code example

ubuntu 18.04 = xenial
ubuntu 20.04 = focal
ubuntu 22.04 = jammy
debian 9 = stretch
debian 10 = buster
debian 11 = bullseye

we can find the distribution code with the command

`lsb_release -sc`